studio/src/App.jsx

```diff
@@ -53,18 +73,12 @@ const App = ({ location, history }) => {
   const onCodeUpdated = React.useCallback(
     (codeData) => {
       if (selected) {
-        let { code, data } = codeData;
```

Comment:
are you sure this should be removed ?

Reply:
Yes!

Reply:
but there are some handling here if any of code or data is undefined, which I believe was needed when typing in the code editor after switching examples. But I don't remember now :) ... just make sure you test properly.

Reply:
The reason is that I add extra properties to codeData, e.g. dataSource (which specifies if data is from QlikEngine or not), appId, sheetId, objectId.
So to avoid many checkings like
if (typeof code === 'undefined') {
          code = selected.code;
}
I use {...selected, ...codeData}